src/neo4j_graphrag/embeddings/cohere.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, model: str = "", **kwargs: Any) -> None:
         if cohere is None:
             raise ImportError(
                 "Could not import cohere python client. "
-                "Please install it with `pip install cohere`."
+                'Please install it with `pip install "neo4j-graphrag[cohere]"`.'
```

Comment:
I find it very confusing to have double quotes on the first line and single quotes on the second one, but that's a ruff thing, it seems to prefer double quotes whenever it's possible (and it's not possible on the second line because we have to use double quote for shell commands (well, I think?))

Reply:
Will a multiline string work? e.g.

```
"""Could not import cohere python client. 
Please install it with `pip install "neo4j-graphrag[cohere]"`."""
```